docs/services/navigation/_index.md

```diff
@@ -0,0 +1,324 @@
+# SMEs: Raymond
+---
+
+The Navigation service is the stateful definition of Viam's [Motion Service](/services/motion/).
+It uses GPS to autonomously navigate a rover [base](/components/base/) to user defined endpoints called `Waypoints`.
+Once these waypoints are added and the mode of the service is [set to `waypoints`](#setmode), the service begins to define the robot's path.
+
+## Configuration
+
+You must configure a [base](/components/base/) with a [movement sensor](/components/movement-sensor/) as part of your robot to configure a Navigation service.
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Services** subtab and navigate to the **Create service** menu.
+Select the type `Navigation` and enter a name for your service.
+
+Click **Create service**:
+
+![An example configuration for a Navigation service in the Viam app Config Builder.](img/navigation-ui-config.png)
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "name": "your-navigation-service",
+    "type": "navigation", 
+    "attributes": {
+        "store": {
+            "type": "memory"
+        },
+        "movement_sensor": "test_movement",
+        "base": "test_base"
+    }
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "name": "test_navigation",
+    "type": "navigation",
+    "attributes": {
+        "store": {
+            "type": "memory"
+        },
+        "movement_sensor": "test_movement",
+        "base": "test_base",
+        "obstacles":[{
+            "geometries": [{
+                "label":"aLabel",
+                "orientation":{
+                    "type":"ov_degrees",
+                    "value":{
+                        "x":1,
+                        "y":0,
+                        "z":0,
+                        "th": 90
+                    }
+                },
+                "x":10,
+                "y":10,
+                "z":10,
+                "translation":{
+                    "x":1,
+                    "y":1,
+                    "z":1
+                }
+            }],
+            "location":{
+                "latitude":1,
+                "longitude":1
+            }
+        }]
+    }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Next, add the JSON `"attributes"` you want the service to have.
+The following attributes are available for `Navigation` services:
+
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `store` | obj | **Required** | The configuration to use for the Navigation data storage. There are two parameters: `type` (`"memory"` or `"mongodb"`) and `config`. <br> Examples: `"store": [{"type": "memory"}]`, `"store": [{ "type": "mongodb", "config": {"uri": "mongodb://127.0.0.1:27017"}]` |
```

Comment:
@npentrel want to clear up here-- Do you have any suggestions for how we can make the formatting here look prettier without using <pre>? From what I can tell there is no default value. I tried looking around but don't have a better description here atm, don't think Raymond did either